src/engine/sim_engine/engine.rs

```diff
@@ -231,32 +231,15 @@ mod tests {
     }
 
     #[test]
-    #[ignore]
-    /// Creating a new pool identical to the previous should succeed
-    fn create_new_pool_twice() {
-        let name = "name";
-        let mut engine = SimEngine::default();
-        engine.create_pool(name, &[], None).unwrap();
-        assert!(match engine
-            .create_pool(name, &[], None)
-            .ok()
-            .and_then(|uuid| uuid.changed())
-        {
-            Some(uuid) => engine.get_pool(uuid).unwrap().1.blockdevs().is_empty(),
-            _ => false,
-        });
-    }
-
-    #[test]
-    /// Creating a new pool with the same name and arguments should return identity
+    /// Creating a new pool with the same name and arguments should return
+    /// identity.
     fn create_pool_name_collision() {
         let name = "name";
         let mut engine = SimEngine::default();
-        engine
-            .create_pool(name, &[Path::new("/s/d")], None)
-            .unwrap();
+        let devices = [Path::new("/s/d")];
```

Comment:
I created a testing macro that you could use here if that's of interest. Since it's already merged, it might make sense to use that instead. It's located [here](https://github.com/stratis-storage/stratisd/blob/develop-2.0.1/src/engine/macros.rs#L157-L162).

Reply:
> I created a testing macro that you could use here if that's of interest. Since it's already merged, it might make sense to use that instead. It's located [here](https://github.com/stratis-storage/stratisd/blob/develop-2.0.1/src/engine/macros.rs#L157-L162).

Probably best to hit a few more missed spots...